drivers/synopsys/ufs/dw_ufs.c

```diff
@@ -97,10 +97,20 @@ static int dwufs_phy_set_pwr_mode(ufs_params_t *params)
 	int result;
 	unsigned int data, tx_lanes, rx_lanes;
 	uintptr_t base;
+	int flags;
 
 	assert((params != NULL) && (params->reg_base != 0));
 
 	base = params->reg_base;
+	flags = params->flags;
+
+	if(flags & UFS_FLAGS_VENDOR_SKHYNIX) {
+		NOTICE("ufs: H**** device must set VS_DebugSaveConfigTime 0x10\n");
+		/* VS_DebugSaveConfigTime */
+		ufshc_dme_set(0xd0a0, 0x0, 0x10);
```

Comment:
ufshc_dme_set returns int (error code), but this is not being checked here - should it be? Similar comment to line 112.

Calls to ufshc_dme_set in lines 126 onwards to check the return value.